.github/workflows/github-release.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: GitHub Release
+
+on:
```

Comment:
You can't use `on` in a reusable action. With what's here I doubt there's a point in making a reusable action out of this: your dependency already does that (except checkout). Modulesync may be a better place to do this.